packages/mars-theme/src/components/index.js

```diff
@@ -13,8 +13,12 @@ import Loading from './loading';
 import Title from './title';
 import PageError from './page-error';
 
+import TwittTooltipCss from './twitt-tooltip/tooltip.css';
```

Comment:
What is the reason for global styles here? I see you have styled component declared inside the component file?